internal/app/render.go

```diff
@@ -65,6 +65,9 @@ func (a *App) renderDir(inputDirpath string, outputDirpath string, excluded []st
 		var output io.Writer = os.Stdout
 		var closer func()
 		if len(outputDirpath) > 0 {
+			if slices.Contains(excluded, filepath.Join(inputDirpath, relPath)) {
```

Comment:
we check this in `renderFile` below, why is it needed here?
maybe you should remove it from `renderFile`?